### PR TITLE
New Port Mythos

### PR DESCRIPTION
**Pull request by Fraxinus88**
# New Port for Mythos

## Game Information
- **Title**: Mythos
- **URL**: https://alkaliii.itch.io/mtgl

## Submission Requirements

CFW Tests:
[X] AmberELEC
[X] ArkOS
[ ] JELOS 
[X] ROCKNIX

Architecture
[X] armHF
[X Panfrost (Mainline) 

Resolution Tests
Test all major resolutions:
[X] 480x320
[X] 640x480
[X] 720x720 (RGB30)
[X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/mythos/mythos/LICENSE.GAME.txt
Comment (kloptops) [inline ports/mythos/mythos/LICENSE.GAME.txt:1]: Is this file meant to be included with this port?

File: ports/mythos/port.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "version": 3,
```
Comment (kloptops) [inline ports/mythos/port.json:2]: version 3?

**Review by kloptops — CHANGES_REQUESTED** [review]
Looks good otherwise?

File: ports/mythos/mythos/LICENSE.GAME.txt
Comment (Fraxinus88) [reply ports/mythos/mythos/LICENSE.GAME.txt:1]: Yes, got a tentative approval, have included it as the LICENSE.aiwi.txt

Comment (kloptops) [reply ports/mythos/mythos/LICENSE.GAME.txt:1]: Yeah but the LICENSE.GAME.txt is for bananamanguy?

**Review by kloptops — CHANGES_REQUESTED** [review]

**Review by Cebion — APPROVED** [review]

**Pull request merged**